### ISSUE TITLE
This project still active?

### ISSUE THREAD

**Issue by conao3** [larstvei/ox-gfm#44]
Hi!
Thanks for a such great package!

The last commit in 2017 and there is no response to some issues and PR.
Anyone who has push right handles these issues and PR?

**Comment by phye** [comment]
Same concern here...

**Comment by kickingvegas** [comment]
Same.

**Comment by larstvei** [comment]
Hi!

I'm sorry, I have not addressed any issue here for a very long time, so the honest answer would be "no, the project is dead".

I'd like it to stay alive though, so I will try to address the issues this week, and I would be open to providing someone else push rights in order to avoid this happening again.

**Comment by conao3** [comment]
You can add me as a collaborator or you can transfer this repository to https://github.com/emacsorphanage maintaining MELPA maintainer @tarsius and community.

**Comment by tarsius** [comment]
Nowadays, if there is a volunteer already, I prefer to not move the package to the orphanage. @larstvei, giving @conao3 commit access, or even transferring the repository to him, seems like a good idea.

**Comment by conao3** [comment]
Allow me to share something embarrassing.
I recently checked this repository to use this particular package after a long time.
When I did, I noticed there was a Collaborator invitation for this repository.
However, the invitation was displayed as expired, preventing me from becoming a Collaborator.

To be honest, I no longer actively develop Emacs plugins, and even if I accepted this invitation, I wouldn't be able to maintain active involvement.
Is there anyone else who would be interested in taking on this maintenance role?
